shopify/shopify_custom_pixel/cl_custom_pixel_code.js

```diff
@@ -0,0 +1,410 @@
+//CustomerLabs Tracking code - Custom pixel code start
```

Comment:
@thisissathish  please refer this __DL__ var and use the fb_skip and debug options  https://github.com/customerlabs/customerlabs_js_helpers/blob/70fe9d03c265663ef6c13af82397ea69fc2b9b5c/shopify/cl_default_ecommerce_events.liquid#L21